cvmfs/sync_item.cc

```diff
@@ -263,7 +263,18 @@ void SyncItem::CheckMarkerFiles() {
 }
 
 void SyncItem::CheckCatalogMarker() {
```

Comment:
Finally, there is no need of declaring a new PathExists() function since the functionality of checking the path is already implemented in StatGeneric().